packages/core/util/Base1DViewModel.ts

```diff
@@ -92,7 +92,7 @@ const Base1DView = types
     pxToBp(px: number) {
       const bp = (self.offsetPx + px) * self.bpPerPx
       let bpSoFar = 0
-      let r = self.displayedRegions[0]
+      let r = getSnapshot(self.displayedRegions[0])
```

Comment:
possibly this change should be applied in linear-genome-view also? was there any typescript warning that alerted this?

Reply:
This is due to needing to generate typescript declarations (see my other comment). When it was spreading the raw region state tree node below this, it was including some internal MST stuff that isn't exported, so there was a typescript error that the internal stuff couldn't be included in the type declarations.

Since linear-genome-view doesn't need to generate type declarations, it's fine for now, but we'll have to change it if we ever want to generate type declarations for it.